storybookjs/storybook
# Fix Incorrect Polymer Link

Issue: Link to Polymer in Supported Frameworks is linked to Angular page
 
## What I did
Changed link for polymer app in README from "/app/agular" to "/app/polymer"

## How to test
Just click it

Is this testable with jest or storyshots?
No

Does this need a new example in the kitchen sink apps?
No

Does this need an update to the documentation?
No

If your answer is yes to any of these, please make sure to include it in your PR.
